src/ObjectMultiplex.ts

```diff
@@ -0,0 +1,93 @@
+import { Duplex } from 'readable-stream';
+import eos from 'end-of-stream';
+import once from 'once';
+import { Substream } from './Substream';
+
+const IGNORE_SUBSTREAM: Record<string, unknown> = {};
```

Comment:
We could use a `Symbol` here instead. That feature just didn't exist when this module was written, I think:
```suggestion
const IGNORE_SUBSTREAM = Symbol('IGNORE_SUBSTREAM');
```